src/Explorer.vue

```diff
@@ -7,7 +7,83 @@
 
 <script>
 import { hex2Bytes } from './utils'
-import { SkipchainRPC } from '@dedis/cothority/skipchain'
+import { SkipchainRPC, SkipBlock } from '@dedis/cothority/skipchain'
+
+const { localStorage } = window
+
+// Defines how many entries the local storage should have at maximum.
+const STORAGE_THRESHOLD = 10
+
+const REGEX_SKIPCHAIN_ID = /^[0-9a-f]+$/
+
+/**
+ * Encode the blocks as a string and store them at the given index which
+ * is the ID of the skipchain
+ *
+ * @param {string} id               The ID of the skipchain
+ * @param {Array<SkipBlock>} blocks The list of blocks to store
+ */
+function storeBlocks (id, blocks) {
+  if (localStorage.length > STORAGE_THRESHOLD) {
+    let minLength = Number.MAX_SAFE_INTEGER
+    let minIndex = -1
+    for (let i = 0; i < localStorage.length; i++) {
```

Comment:
Am I understanding well?

```suggestion
    // We iterate over all the elements of the local storage that have a key
    // corresponding to a skip block id and remove the element that holds
    // the smallest piece of data.
    for (let i = 0; i < localStorage.length; i++) {
```

Reply:
Except each element in the local storage has a key. You have access to the length and a function to get the key at a given index which is the minimum to iterate over the stored elements.

Reply:
Ok, thanks. I would add a comment that briefly describe what's happening in this loop. I would also specify that this method removes the smallest element in case there is more than STORAGE_THRESHOLD elements and not all elements have a length greater or equal than Number.MAX_SAFE_INTEGER.

Tell me if I am wrong, but anyone that later uses local storage with an hexadecimal key can expect the content to be deleted, right? If that's the case, I would also comment it somewhere.